main.go

```diff
@@ -18,7 +18,7 @@ import (
 
 //go:generate go-bindata -nometadata -pkg templatebin -o templatebin/bindata.go templates templates/singleton templates_test templates_test/singleton
 
-const sqlBoilerVersion = "3.4.0"
+const sqlBoilerVersion = "3.4.1"
```

Comment:
Version changes are done at release time and since this adds a feature it'd be a minor version not a patch :) Please remove

Reply:
Done.